app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,7 @@
     package="chat.literacyapp.org.chat">
```

Comment:
This is a previous error, but we need to update the package name to "org.literacyapp.chat" 

@adrygll2 Could you handle this?